includes/api/class-wc-rest-coupons-controller.php

```diff
@@ -330,6 +377,11 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit' ),
 				),
+				'date_expires_gmt' => array(
```

Comment:
Now have 2 `UTC` dates here.
Better change to:

`date_expire` = `The date the coupon expire, in the site's timezone.`
`date_expire_gmt` = `The date the coupon expire, as GMT.`